learning-another-language/vscode-and-style.md

```diff
@@ -253,20 +254,20 @@ Both the `package.json` and `package-lock.json` files _should_ be checked into v
 
 We can also add dependencies that will only be used during development with `npm install <dependency> --save-dev`. Development dependencies typically include things like linters, code formatters, and testing libraries.
 
-For example, we can add the Eslint library, along with a few plugins, with:
+For example, we can add the Eslint library, along with plugins to support linting tests and code written with the React framework, using the command:
 
 ```bash
-$ npm install eslint eslint-config-standard eslint-plugin-import eslint-plugin-node eslint-plugin-react --save-dev
+$ npm install eslint eslint-plugin-import eslint-plugin-jest eslint-plugin-node eslint-plugin-promise @eslint/eslintrc @eslint/js globals --save-dev
 ```
 
 Notice the version information for the development dependencies is added to the `package.json` file under the `devDependencies` key.
 
-To use Eslint, we need to set up a `.eslintrc.json` file to provide the rules we want to enforce.
+To use Eslint, we need to set up a `eslint.config.mjs` file to provide the rules we want to enforce.
 
 In the project directory type:
```

Comment:
I'm confused by the wording that's already in the lesson. 

It says "In the project directory type:" 

Does this mean, "In your terminal, when you're current working directory is the root of your project directory, run the following command." ?

Reply:
Yes, that is the intent. I'll update the line for clarity!